vspec/loggingconfig.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+#
+# Convert vspec files to various other formats
+#
+
+import logging
+
+def initLogging():
+    """
+    Initialize logging
+    """
+    logging.basicConfig(level=logging.DEBUG, format='%(levelname)-8s %(message)s')
```

Comment:
For now I assume DEBUG is OK as default level. But do we want to keep  it as default in the long run, or should rather INFO be the default level?

Reply:
Yes. a command line arg is pretty common.